frontend/src/app/libs/common-components-demo/components/button-demo/button-demo.component.html

```diff
@@ -22,6 +22,17 @@
 				<app-button type="flat" [disabled]="true">Button</app-button>
 			</app-demo-state>
 		</div>
+		<div class="button-demo__active">
+			<app-demo-state [title]="'medium icon'" class="button-demo__container">
+				<app-button [icon]="'done'" [iconSize]="iconSizes.M"></app-button>
```

Comment:
you can pass it as string `[size]="extra-small"`

Reply:
also update input to [size] no need to mention word icon since we already in icon context